### PR TITLE
[QM-6811] allow form submissions in client frame by default

### PR DESCRIPTION
**Pull request by jakenewby**

### CODE REVIEW

**Comment by MattCheely** [comment]
@jakenewby Can you run `npm version patch` and push that commit? The build doesn't auto-bump versions yet.

**Comment by jakenewby** [comment]
^ done

File: src/FrameManager.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ class FrameManager {
 
     this._iframe.setAttribute(
       'sandbox',
-      'allow-scripts allow-same-origin allow-modals'
+      'allow-scripts allow-same-origin allow-modals allow-forms'
```
Comment (justinaray) [inline src/FrameManager.ts]: I think the allow-forms is pretty safe to add here unconditionally, so I'm fine with this.

It does bring up a broader question if some of the more risky (allow-scripts, allow-same-origin, allow-top-navigation, etc.) and more UX-affecting (allow-modals, allow-popups, etc.) switches should be opt-in.  We could allow the host to specify these or try to make educated decisions based on if the iframe is from the same origin or not.

Comment (jakenewby) [reply src/FrameManager.ts]: agreed. I think, optimally, we would provide configuration options per-client with some sane fallbacks (just "allow-scripts", probably).

**Pull request merged**